src/styles/_animatedscroll.scss

```diff
@@ -0,0 +1,196 @@
+$topOffset: 64px; // sticky header height
```

Comment:
This is already in `_variables.scss` as `$sticky-header-height`, if you want to consolidate.

Reply:
thanks, updated